<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui_TeamEdit</class>
 <widget class="QDialog" name="Gui_TeamEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Team list</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/team_dialog_16</normaloff>:/icons/team_dialog_16</iconset>
  </property>
  <layout class="QVBoxLayout" name="centralLayout">
   <item>
    <widget class="QListView" name="teamList">
     <property name="toolTip">
      <string>List of all participating teams</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="addRemoveTeamLayout">
     <item row="1" column="0">
      <widget class="QPushButton" name="addTeamButton">
       <property name="toolTip">
        <string>Click to add a new team</string>
       </property>
       <property name="text">
        <string>Add team</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/team_add_16</normaloff>:/icons/team_add_16</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="removeTeamButton">
       <property name="toolTip">
        <string>Click to remove current team</string>
       </property>
       <property name="text">
        <string>Remove team</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/team_remove_16</normaloff>:/icons/team_remove_16</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="editTeamButton">
       <property name="toolTip">
        <string>Click to edit current team</string>
       </property>
       <property name="text">
        <string>Edit team</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/team_edit_16</normaloff>:/icons/team_edit_16</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDockWidget" name="addEditWidget">
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="features">
      <set>QDockWidget::DockWidgetClosable</set>
     </property>
     <property name="windowTitle">
      <string>Add/Edit Team</string>
     </property>
     <widget class="QWidget" name="dockWidgetContents">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="addEditLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="teamNameLabel">
           <property name="text">
            <string>Team name</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="3">
          <widget class="QLineEdit" name="teamNameEdit">
           <property name="toolTip">
            <string>Type in team's name</string>
           </property>
           <property name="placeholderText">
            <string>Type team name here</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QTimeEdit" name="finishTimeEdit">
           <property name="toolTip">
            <string>Change team's finish time</string>
           </property>
           <property name="currentSection">
            <enum>QDateTimeEdit::HourSection</enum>
           </property>
           <property name="displayFormat">
            <string>HH:mm</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="finishTimeLabel">
           <property name="text">
            <string>Finish time</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QPushButton" name="doneButton">
           <property name="toolTip">
            <string>Click when done editing team</string>
           </property>
           <property name="text">
            <string>Done</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/icons/ok_16</normaloff>:/icons/ok_16</iconset>
           </property>
          </widget>
         </item>
         <item row="3" column="2" colspan="2">
          <widget class="QPushButton" name="cancelButton">
           <property name="toolTip">
            <string>Click to abort editing</string>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/icons/cancel_16</normaloff>:/icons/cancel_16</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="teamMembersLabel">
           <property name="text">
            <string>Members</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="teamMembersEdit">
           <property name="toolTip">
            <string>Change number of team members</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="closeButton">
     <property name="toolTip">
      <string>Click to close team dialog</string>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icons/close_16</normaloff>:/icons/close_16</iconset>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
